src/index.js

```diff
@@ -1,10 +1,40 @@
 const { Telegraf } = require("telegraf");
 require("dotenv").config();
+const Groq = require("groq-sdk");
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
+const groq = new Groq();
 
 bot.start((ctx) => ctx.reply("Welcome to your AI Assistant!"));
 
+// Handle messages and respond using Groq SDK
+bot.on("text", async (ctx) => {
```

Comment:
**praise**: Excellent use of the Telegraf library to handle text messages. This is a clean and efficient way to manage incoming user messages.